source/declarations.tex

```diff
@@ -6972,12 +6972,11 @@
 An \grammarterm{enumerator-definition} with \tcode{=} gives the associated
 \grammarterm{enumerator} the value indicated by the
 \grammarterm{constant-expression}.
-If the first \grammarterm{enumerator}
-has no \grammarterm{initializer}, the value of the corresponding constant
-is zero. An \grammarterm{enumerator-definition} without an
-\grammarterm{initializer} gives the \grammarterm{enumerator} the value
-obtained by increasing the value of the previous \grammarterm{enumerator}
-by one.
+An \grammarterm{enumerator-definition} without \tcode{=} gives the associated
```

Comment:
Can we quote the entire grammar snippet here, e.g. "An enumerator-definition without = _constant-expression_ ..."?

Otherwise, a twisted soul might try to circumvent this by putting "a=b" inside the constant-expression, given a suitably overloaded operator=.

Reply:
Uh, I just noticed the "with =" in the preceding sentence, so it seems "without =" is fine as-is.